temporal-sdk/src/main/java/io/temporal/worker/WorkerOptions.java

```diff
@@ -385,24 +387,25 @@ public Builder setDisableEagerExecution(boolean disableEagerExecution) {
 
     /**
      * Opts the worker in to the Build-ID-based versioning feature. This ensures that the worker
-     * will only receive tasks which it is compatible with. For more information see: TODO: Doc link
+     * will only receive tasks which it is compatible with.
      *
      * <p>Defaults to false
      */
     @Experimental
+    @Deprecated // TODO REVIEW: Are we actually deprecating these yet?
```

Comment:
Not entirely sure if we are deprecating these or not

Reply:
I think we can, though I think we should commit to deprecating all past versioning things I think